packages/babel-helper-compilation-targets/test/__snapshots__/targets-parser.spec.js.snap

```diff
@@ -2,13 +2,12 @@
 
 exports[`getTargets esmodules returns browser supporting modules and keyed browser overrides 1`] = `
 Object {
-  "android": "61.0.0",
+  "android": "81.0.0",
   "chrome": "61.0.0",
   "edge": "16.0.0",
   "firefox": "60.0.0",
   "ie": "11.0.0",
   "ios": "10.3.0",
-  "node": "13.2.0",
   "opera": "48.0.0",
```

Comment:
In theory this shouldn't have changed, but:
- `node` is removed because Browserslist uses `compat-table` which doesn't include data for Node.js. It's not a problem, because all the transforms required for Node.js 13.2 are required for the other browsers in this query anyway :shrug: 
- `android` is changed because `compat-table` only keeps data for the last version. I kind of expected Browserslist to automatically use the Chrome version (like we do in our generated compat data) since we are using the `mobileToDesktop: true` Browserslist option, but again it's not observable since `android 61` is already covered by `chrome 61` in these targets.

Reply:
This will become a breaking change, if we decide `{ esmodule: true }` should _intersect_ other queries (https://github.com/babel/rfcs/pull/2#issuecomment-714785228), i.e. `B({ esmodule: true }) ∩ B({ android: 78 })` will no longer be `B({ android: 78 })` but `B({ android: 81 })`. Likewise for `node`.

@ai I think it is a bug if `supports feature` query does not respect `mobileToDesktop: true`
```
npx browserslist --mobile-to-desktop "supports es6-module"
```
should contain `android 61`.

Reply:
I realized that we won't be able to use a Browserslist-based intersection anyway.

Currently `targets: ["chrome 30"]` resolves to "compile everything down to chrome 30, but `targets: ["chrome 30 and supports es6-module"]` resolves to an empty set because there isn't any browser that both _is chrome 30_ and _supports esm_.

The query should have been `chrome >= 30 and supports es6-module`, but it given how `targets` has always being working users probably rely on `>=` being optional.

Reply:
Reverted